packages/sdk/src/schema/pages.ts

```diff
@@ -129,8 +129,20 @@ export const ProjectNewRedirectPath = PagePath.or(
   }, "Must be a valid URL")
 );
 
+export const ProjectOldRedirectPath = PagePath.or(
+  z
+    .string()
+    .refine(
+      (path) =>
+        new RegExp(/^(?:\/[-_a-zA-Z0-9*:?\\\p{L}\\p{N}\\p{M}\\\/]+)+$/gmu).test(
```

Comment:
is there an original implementation of this? I think its a non-trivial thing to write and we should be able to use something battle-tested

Reply:
I just extended `\\p{L}\\p{N}\\p{M}\\` to the original one to allow non-latin characters though

Reply:
Btw is not possible to use any of the standard api for this? e.g. URL api

Reply:
Will see, ideally should work though. Will check about the `+/-`'s of both approaches.

Reply:
> where is the original coming from? is it worth linking to?

Original regex is from `PagesPath`, just extending with latin characters. Until there is a decision to support non-latin across all the paths inside webstudio projects. Maybe checking only for old paths (projects that are designed in another platforms)  would be safe way to go.